### PR TITLE
[VEN-2741]: Bridge configurations on zksync mainnet 

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves VEN

### CODE REVIEW

**Comment by chechu** [comment]
We could add to the same PR the TX (and VIP for bscmainnet) to be executed on the remote networks to configure the trustworthiness relationship with zkSync

**Review by 0xSolDev — APPROVED** [review]

File: multisig/proposals/zksyncmainnet/vip-002/index.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setWhitelist(address,bool)",
+      params: [zksyncmainnet.GUARDIAN, true],
+    },
+    {
+      target: XVS,
+      signature: "setMintCap(address,uint256)",
+      params: [XVS_BRIDGE_DEST, XVS_MINT_LIMIT],
+    },
+  ]);
```
Comment (chechu) [inline multisig/proposals/zksyncmainnet/vip-002/index.ts:199]: Don't we have to configure the limits between zkSync and the rest of the networks? I think we have to invoke the following functions on the `XVS_BRIDGE_ADMIN_PROXY`, for each network:

* setMaxDailyLimit(uint16,uint256)
* setMaxSingleTransactionLimit(uint16,uint256)
* setMaxDailyReceiveLimit(uint16,uint256)
* setMaxSingleReceiveTransactionLimit(uint16,uint256)

Comment (chechu) [reply multisig/proposals/zksyncmainnet/vip-002/index.ts:199]: I see these functions were invoked during the deployment. Ignore this comment

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/358

**Pull request merged**